azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -1091,6 +1139,18 @@ public Props fetchProjectProperty(Project project, String propsName)
     }
   }
 
+  public Set<Integer> fetchProjectIdsByEventType(EventType eventType) throws ProjectManagerException {
+    QueryRunner runner = createQueryRunner();
```

Comment:
Don't we need connection here like you did in fetchAllInactiveProjects?

Reply:
This runner.query function is internally preparing a connection, we can use that in all inactive projects fetch also but that I used same as fetchAllActive, that's why used connection there